packages/core/src/percy.js

```diff
@@ -321,6 +325,18 @@ export class Percy {
     }.call(this));
   }
 
+  // Driver Wrapper
+  async driverWrapper(options) {
+    this.sessionId = options.sessionId;
+    this.commandExecutorUrl = options.commandExecutorUrl;
+    this.capabilities = options.capabilities;
+    this.snapshotName = options.snapshotName;
```

Comment:
Just shifting up in order to keep simple strings together and complex jsons together.
```suggestion
    this.snapshotName = options.snapshotName;
    this.commandExecutorUrl = options.commandExecutorUrl;
    this.capabilities = options.capabilities;
```

Reply:
Moving `snapshotName` out of constructor, so this will get updated.

Reply:
Done.